Assets/Mirror/Authenticators/TimeoutAuthenticator.cs

```diff
@@ -37,13 +39,13 @@ public override void OnServerAuthenticate(NetworkConnection conn)
 
         IEnumerator BeginAuthentication(NetworkConnection conn)
         {
-            if (LogFilter.Debug) Debug.Log($"Authentication countdown started {conn} {timeout}");
+            if (LogFilter.Debug) logger.Log($"Authentication countdown started {conn} {timeout}");
```

Comment:
Can you replace all of the
`if (LogFilter.Debug)`
with
`if (logger.LogEnabled()) `

Reply:
fixed below